runtime/lua/vim/pack.lua

```diff
@@ -875,11 +903,11 @@ function M.update(names, opts)
     end
 
     -- Fetch
-    -- Using '--tags --force' means conflicting tags will be synced with remote
-    git_cmd(
-      { 'fetch', '--quiet', '--tags', '--force', '--recurse-submodules=yes', 'origin' },
-      p.path
-    )
+    if not opts.__offline then
```

Comment:
why is double-underscore used here? single underscore means "private don't use". there is no other "super private" variation...

Reply:
There are cases of using double-underscore to mean "super private" and I remember to have seen that in Matrix discussions. Using single underscore is indeed easier, I agree.